.github/stale.yml

```
@@ -28,13 +25,13 @@ staleLabel: stale
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
   This pull request has been automatically marked as stale because it
-  has not had recent activity. It will be closed if no further
-  activity occurs.
-
+  has not had any activity for 60 days. It will be closed if no further
+  activity occurs for another 30 days.
+  
   If you are a maintainer or core committer, please follow-up on this
   issue to identify what steps should be taken by the author to move this
```

IIRC somebody suggested to replace the term "issue" here:
```suggestion
  pull request to identify what steps should be taken by the author to move this
```